### DISCUSSION TITLE
How to display previous and next slide ?

### DISCUSSION THREAD

**Discussion by lucasriondel** [nolimits4web/swiper#5785] [Q&A]
Hi everyone ! 

I'm trying to achieve this. 
![image](https://user-images.githubusercontent.com/7773454/172815782-688dd386-b7c3-4e76-8484-0acc9535c968.png)

I am using react-swiper. My slide elements must be 294px in width, with a gap of 50 between them. Distance between card border and screen border must be 70px minimum, and increasing if the window size increases (content always centered). 

I am having a lot of trouble trying to achieve this. Just setting the width of my slide seems to confuse Swiper heavily and breaks everything. 

Does someone know how I can achieve this easily ? 

Thanks in advance for the answer.

**Comment by Arhell** [comment]
https://stackoverflow.com/questions/71203586/swiper-js-how-to-show-2-half-slides-and-2-full-slides

**Comment by SvenBudak** [comment]
Is there no clean way to do that by options only?
